src/CsFixerConfig.php

```diff
@@ -54,6 +54,7 @@ class CsFixerConfig extends Config
         'function_typehint_space' => true,
         'lowercase_cast' => true,
         'method_argument_space' => ['on_multiline' => 'ensure_fully_multiline'],
+        'modernize_strpos' => true,
```

Comment:
Not compatible with PHP before 8.0. Can be merged once PHP support is raised. See also #49